### PR TITLE
Dbz 8090 incorrect signal data collection formats in signaling and snapshots docs

### PR DESCRIPTION
**Pull request by roldanbob**
[DBZ-8090](https://issues.redhat.com/browse/DBZ-8090)

Updates the incremental snapshot documentation to correct the format for specifying fully qualified table names in snapshot signals. A relatively simple content change that's made complex due to the many moving parts. This PR includes the  following changes:
 
* Refactors the existing shared files to reference new snippet files that host examples with the appropriate FQ names for each context.   
* Edits content to clarify the use of multiple signaling channels and place information about the proper naming formats in-line vs. relying on cross-references to descriptions in configuration properties.
* Defines new attribute `collection-container` to identify the container used to form the FQ collection name for each DB. 
* Adds missing `connector-name` attributes for Spanner and Vitess connectors to eliminate bothersome build errors that resulted from unresolved attributes.
* Removes `{context}` attributes from `signaling.adoc` where value cannot be resolved.
* Moves content from shared files in the `partials` directory into the file `proc-running-an-ad-hoc-snapshot-with-additional-conditions.adoc` to eliminate headings and anchor IDs that are invalid for use in shared files.
* Add description of why a user might want to stop an incremental snapshot.
* Edit the note about how to list table names that include a dot (`.`) to improve clarity.  
* Remove the embedded topic _Pass-through database schema history properties for configuring producer and consumer clients_ from the upstream `ref-connector-pass-through-database-driver-configuration-properties.adoc` file and add the content to the new file, `rec-connector-pass-through-database-history-properties-for-configuring-producer-and-consumer-clients`.
* Add  heading for the _Add  heading for topic "Pass-through database schema history properties for configuring producer and consumer clients_ topic to configuration properties section for each connector 
* Apply  consistent order to subheadings in the configuration properties sections and to the "mini TOC" bulleted list of configuration properties that begins each section.
* Edits description of `kafka.consumer.offset.commit.enabled property` introduced in DBZ-7164.

Tested in a local Antora build and in a local downstream build.

### CODE REVIEW

File: documentation/modules/ROOT/partials/modules/all-connectors/proc-running-an-ad-hoc-snapshot-with-additional-conditions.adoc [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+
+[source,sql,subs="+attributes,+quotes"]
+----
+SELECT * FROM _<data-collection>_ WHERE _<filter>_ ....
+----
+
+The following example shows a SQL query to send an ad hoc incremental snapshot request with an additional condition to the signaling {data-collection}:
+[source,sql,indent=0,subs="+attributes,+quotes"]
+----
+INSERT INTO _<signalTable>_ (id, type, data) VALUES (_'<id>'_, _'<snapshotType>'_, '{"data-collections": ["_<tableName>_","_<tableName>_"],"type":"_<snapshotType>_","additional-conditions":[{"data-collection": "_<tableName>_", "filter": "_<additional-condition>_"}]}');
```
Comment (Naros) [inline documentation/modules/ROOT/partials/modules/all-connectors/proc-running-an-ad-hoc-snapshot-with-additional-conditions.adoc:20]: Should this example also make use of the `collection.container` variable?

**Review by Naros — COMMENTED** [review]
Hi @roldanbob, LGTM except one open question, see below.

File: documentation/modules/ROOT/partials/modules/all-connectors/proc-running-an-ad-hoc-snapshot-with-additional-conditions.adoc (hunk 1 above)
Comment (roldanbob) [reply documentation/modules/ROOT/partials/modules/all-connectors/proc-running-an-ad-hoc-snapshot-with-additional-conditions.adoc:20]: I think that if you read it in context, it becomes more clear that the example that you reference shows the general format for a query, using `signalTable` and `tableName` variables (as well as `snapshotType` and `additional-condition`) in place of actual values. The followup examples then show an actual query that replaces the variables  with dummy values, such as `myschema.debezium_signal` and `"schema1.table1", "schema1.table2"`. This mirrors the [approach that's used in the published documentation](https://debezium.io/documentation/reference/2.7/connectors/db2.html#db2-incremental-snapshots-additional-conditions). 
I suppose that I could make it more clear by explicitly stating up front that the first example shows the general format, and maybe making the variable names more descriptive (e.g., `fullyQualifiedTableName`).  WDYT?

Comment (Naros) [reply documentation/modules/ROOT/partials/modules/all-connectors/proc-running-an-ad-hoc-snapshot-with-additional-conditions.adoc:20]: In that case, feel free to merge yourself.

**Comment by roldanbob** [comment]
@Naros I know that you previously said that I should merge this myself, but due to the increased volume of changes, I wonder could you give this a sanity check? There are a few relatively minor content edits, mostly regarding property descriptions, but otherwise the changes are primarily related to formatting and refactoring/restructuring to improve build reliability.

**Pull request merged**